monkeys/src/main/kotlin/com/wire/kalium/monkeys/homeDirectory.kt

```diff
@@ -29,13 +29,12 @@ fun coreLogic(
     rootPath: String,
 ): CoreLogic {
     val coreLogic = CoreLogic(
-        rootPath,
-        kaliumConfigs = KaliumConfigs(
+        rootPath, kaliumConfigs = KaliumConfigs(
             developmentApiEnabled = true,
             encryptProteusStorage = true,
             isMLSSupportEnabled = true,
             wipeOnDeviceRemoval = true,
-        ), "Wire Infinite Monkeys"
+        ), userAgent = "Wire Infinite Monkeys", useInMemoryStorage = false
```

Comment:
I think you can use `inMemoryStorage = true` now. After 

- #2274

Reply:
I'll enable it again :) Forgot to push